### PR TITLE
Advertise mdakits

### LINKED ISSUE #320: Link to MDAKits registry
Corresponding to https://github.com/MDAnalysis/mdanalysis/issues/4296

### PR DESCRIPTION
**Pull request by lilyminium**
Fixes #320 

Waiting on mda-sphinx-theme 1.3.0 to hit conda-forge.

----
:books: Documentation preview :books:: https://mdanalysis--321.org.readthedocs.build/en/321/

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -73,7 +73,6 @@ Wherever possible, do not take these conversations to private channels, includin
    examples/quickstart
    faq
    examples/README
-   API docs <https://docs.mdanalysis.org/stable/index.html>
```
Comment (IAlibay) [inline doc/source/index.rst:76]: Does development docs below also want to be there?

**Review by IAlibay — COMMENTED** [review]
In addition to purely linking to the mdakits page, the userguide is in the right place to provide some kind of written "so you're looking to write some code extension, have you considered a kit?" blurb, possibly somewhere in contributing?

File: doc/source/index.rst (hunk 1 above)
Comment (lilyminium) [reply doc/source/index.rst:76]: Thanks, good pickup!

**Comment by lilyminium** [comment]
I added a section on MDAKits to the contribution docs -- would appreciate a general short lookover if possible!

File: doc/source/contributing.rst [hunk 2]
```diff
@@ -32,10 +32,28 @@ Where to start?
...
+If you are new to Git and version control, have a look at
 :ref:`version-control`.
 
+If you are looking to contribute a code or documentation fix, or core feature to MDAnalysis,
+and you are brand new to MDAnalysis or open-source development -- we recommend going
+through the guides for :ref:`contributing to the main codebase <working-with-mdanalysis-code>`
+or the :ref:`user guide <working-with-user-guide>`. 
+
+If you are looking to contribute your own project or new addition to MDAnalysis,
+we encourage you to consider doing so by creating an
```
Comment (IAlibay) [inline doc/source/contributing.rst]: We probably shouldn't block off contributing to the core library completely - here I would say something like "open an issue or consider creating an mdakit"

**Review by IAlibay — COMMENTED** [review]
just the one thing, otherwise lgtm

File: doc/source/contributing.rst (hunk 2 above)
Comment (lilyminium) [reply doc/source/contributing.rst]: Thanks -- changed!

File: doc/source/contributing.rst [hunk 3]
```diff
@@ -32,10 +32,28 @@ Where to start?
...
+If you are new to Git and version control, have a look at
 :ref:`version-control`.
 
+If you are looking to contribute a code or documentation fix, or core feature to MDAnalysis,
+and you are brand new to MDAnalysis or open-source development -- we recommend going
+through the guides for :ref:`contributing to the main codebase <working-with-mdanalysis-code>`
+or the :ref:`user guide <working-with-user-guide>`. 
+
+If you are looking to contribute your own project or new addition to MDAnalysis,
+we encourage you to open an issue or consider creating an
```
Comment (IAlibay) [inline doc/source/contributing.rst]: Maybe add a link to the issue tracker?

**Review by IAlibay — CHANGES_REQUESTED** [review]

File: doc/source/contributing.rst [hunk 4]
```diff
@@ -32,10 +32,28 @@ Where to start?
...
 enhancements, and ideas are welcome.
 
-If you are brand new to MDAnalysis or open-source development, we recommend going
-through the guides for :ref:`contributing to the main codebase <working-with-mdanalysis-code>` or the :ref:`user guide <working-with-user-guide>`. If you are new to Git and version control, have a look at
+If you are new to Git and version control, have a look at
 :ref:`version-control`.
 
+If you are looking to contribute a code or documentation fix, or core feature to MDAnalysis,
+and you are brand new to MDAnalysis or open-source development -- we recommend going
+through the guides for :ref:`contributing to the main codebase <working-with-mdanalysis-code>`
```
Comment (IAlibay) [inline doc/source/contributing.rst:40]: pre-commit is complaining somewhere here

**Review by IAlibay — APPROVED** [review]
cheers

**Review by orbeckst — APPROVED** [review]
Great idea, very good. 

(Sorry, didn’t have time to look at the technical side. Content is an excellent addition!)

**Comment by lilyminium** [comment]
Thanks for the reviews @IAlibay and @orbeckst!

**Pull request merged**